app/models/agents/website_agent.rb

```diff
@@ -166,6 +169,27 @@ def validate_options
       validate_web_request_options!
     end
 
+    def validate_consider_http_success_option!
+      consider_success = options["http_success_codes"]
+      if consider_success != nil
```

Comment:
Yes, I think for ease of use, `[]`, `nil`, `''`, and `false` should all be treated as "valid" and ignored.  If you use `if consider_success.present?` here, I think that would work, because you already have `consider_success.present? &&` used in `consider_response_successful?`.  Does that make sense?